README.md

```diff
@@ -25,22 +25,36 @@ host_key_checking = False
 ```
 cd ~/src/
 git clone git@github.com:redhat-cop/casl-ansible.git
-git clone git@github.com:openshift/openshift-ansible.git
 ```
 * Copy `casl-ansible/inventory/sample.casl.example.com.d/inventory/clouds.yaml` to `~/.config/openstack/clouds.yaml`
 
+* Download/untar `openshift-ansible` for use as part of the install. See table below for versions / urls to be used for the download. Note that other versions may work as well, but these are the ones we have tested and found to be stable. Feel free to submit PRs with updated versions as they are found to be functional. 
```

Comment:
We need to instruct them to unpack the openshift-ansible code in the same directory as casl-ansible. Otherwise they'd need to pass a variable.